Segmentation violation when running `ddev list` with invalid entry in global_config.yaml

**Describe the bug**
I somehow had an invalid entry in the global_config.yaml similar to this one:
```
project_info:
  Test:
    approot: ""
```
Running `ddev list` in my home directory caused a panic:
```
$ ddev list
unable to create project at project root : ddev config is not useful in your home directory (/home/XXX)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x9e9af9]

goroutine 1 [running]:
github.com/drud/ddev/pkg/ddevapp.GetProjects(0xbfcb9659b19b2800, 0x2190f1c, 0x11952a0, 0xc89a3d, 0x9, 0xcdc178)
	/workdir/pkg/ddevapp/utils.go:355 +0x619
github.com/drud/ddev/pkg/ddevapp.List(0x0, 0x1)
	/workdir/pkg/ddevapp/ddevapp.go:123 +0xa5
github.com/drud/ddev/cmd/ddev/cmd.glob..func26(0x1188cc0, 0x11c6870, 0x0, 0x0)
	/workdir/cmd/ddev/cmd/list.go:26 +0x45
github.com/spf13/cobra.(*Command).execute(0x1188cc0, 0x11c6870, 0x0, 0x0, 0x1188cc0, 0x11c6870)
	/workdir/vendor/github.com/spf13/cobra/command.go:846 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1189f20, 0x5f4de1e6, 0x319a241d, 0xc22ad2c278)
	/workdir/vendor/github.com/spf13/cobra/command.go:950 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/workdir/vendor/github.com/spf13/cobra/command.go:887
github.com/drud/ddev/cmd/ddev/cmd.Execute()
	/workdir/cmd/ddev/cmd/root.go:153 +0x3c
main.main()
	/workdir/cmd/ddev/main.go:21 +0xf4
```

**To Reproduce**
1. Create an (additional) project entry with an empty `approot` in the global_config.yaml
2. Start Terminal in home directory
3. Run `ddev list`
4. See error

**Expected behavior**
The list of all projects should be shown (and `ddev config` should not be called).

Additionally it might be good to notify the user on the invalid entry and give an option to delete it or just delete the entry and notify the user that it was deleted?

I did not add this entry manually, so it is possible that this entry is generated by `ddev` also this should be prevented. 

**Version and configuration information (please complete the following information):**

- Host computer OS and Version: Ubuntu 20.04.1
- DDEV-Local version	v1.16.0-alpha2

**Additional context**
Trying to delete the entry with `ddev stop --unlist Test` does not work (also causes a segmentation violation).

--- a/pkg/ddevapp/utils.go
+++ b/pkg/ddevapp/utils.go
@@ -351,13 +351,14 @@ func GetProjects(activeOnly bool) ([]*DdevApp, error) {
 
 		app, err := NewApp(info.AppRoot, true, nodeps.ProviderDefault)
 		if err != nil {
-			util.Warning("unable to create project at project root %s: %v", info.AppRoot, err)
-			app.Name = name
+			util.Warning("unable to create project at project root '%s': %v", info.AppRoot, err)
+			continue
 		}
 
 		// If the app we just loaded was already found with a different name, complain
 		if _, ok := apps[app.Name]; ok {
 			util.Warning(`Project '%s' was found in configured directory %s and it is already used by project '%s'. If you have changed the name of the project, please "ddev stop --unlist %s" `, app.Name, app.AppRoot, name, name)
+			continue
 		}
 
 		if !activeOnly || (app.SiteStatus() != SiteStopped && app.SiteStatus() != SiteConfigMissing && app.SiteStatus() != SiteDirMissing) {

